### DISCUSSION TITLE
Add RIMusic in music provider list

### DISCUSSION THREAD

**Discussion by athome10320** [music-assistant/support#2646] [Feature requests and ideas]
Is it possible to add RiMusic to the list of music providers ?

**Comment by OzGav** [comment]
You should change the title so others can find this and upvote it. Having said that MA already supports YouTube music?

**Reply by athome10320** [reply]
Yes but with youtube or spotify provider it is necessary to have a subscrpition.
Rimusic is free no subscription need.

**Reply by OzGav** [reply]
Then the answer is No. MA will not permit the avoidance of subscription charges or ads.
